src/main/java/rx/internal/util/ObjectPool.java

```diff
@@ -54,11 +49,11 @@ private ObjectPool(final int min, final int max, final long validationInterval)
         // initialize pool
         initialize(min);
 
-        schedulerWorker = Schedulers.computation().createWorker();
```

Comment:
Thanks ... let's review that one separately if you're still interested in pursuing it. I'm not convinced the interference matters, especially since it just moves the interference to the OS scheduler with more threads.